### PR TITLE
windows/logoff: add Indonesian translation

### PR DESCRIPTION
**Pull request by hamexhanif**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/windows/logoff.md
Comment (reinhart1010) [inline pages.id/windows/logoff.md:3]:
```suggestion
> Akhiri sesi masuk.
```
Imperative mood, see #8576.

File: pages.id/windows/logoff.md
Comment (reinhart1010) [inline pages.id/windows/logoff.md:6]:
```suggestion
- Akhiri sesi saat ini:
```

File: pages.id/windows/logoff.md
Comment (reinhart1010) [inline pages.id/windows/logoff.md:10]:
```suggestion
- Akhiri sesi dengan nama atau id-nya:
```

File: pages.id/windows/logoff.md
Comment (reinhart1010) [inline pages.id/windows/logoff.md:14]:
```suggestion
- Akhiri sesi pada server tertentu yang terhubung melalui RDP:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Hi, thanks for contributing! Here are some of my notes to review.

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Review by reinhart1010 — APPROVED** [review]
Thanks! We will need to wait for another maintainer to approve before merging this PR.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**
